### ISSUE TITLE
Under-fitting issue for instance segmentation (masking)

### ISSUE THREAD

**Issue by ahmetozlu** [matterport/Mask_RCNN#2536]
I have been training Mask R-CNN model with my own custom training data. In the data/train folder I have an image with the annotation (via_region_data.json) and I have exactly the same image and same annotation (via_region_data.json) in the data/val folder as well. I have trained the model with that train and val data (contains exactly same image). And, I processed that image with the model which was trained with that image. I expect to see 100% percentage accuracy for both of detection and masking. 

Model has 100% percentage (36 objects I have in the image all of them have bounding boxes) accuracy in detection but it has 86% percentage in masking (36 object I have in the image and just 31 of them have masking). I need to increase the ration of masking in this image to be sure that okay my model is ready to train with mode images to reach super accuracy Do you have any suggestion for this case? Thanks..

**Comment by ahmetozlu** [comment]
I have solved the issue by updating the model configuration. Basically, set USE_MINI_MASK false works a lot.. One of the configuration works best for my case in training is like this:

    # Number of training steps per epoch
    STEPS_PER_EPOCH = 100

    # Skip detections with < 90% confidence
    DETECTION_MIN_CONFIDENCE = 0.9

    USE_MINI_MASK = False

    # Maximum number of ground truth instances to use in one image
    MAX_GT_INSTANCES = 200

    # Max number of final detections per image
    DETECTION_MAX_INSTANCES = 400
